neural_modelling/src/neuron/synapse_row.h

```diff
@@ -232,20 +232,33 @@ static inline weight_t synapse_row_sparse_weight(uint32_t x) {
     return x >> (32 - SYNAPSE_WEIGHT_BITS);
 }
 
+////! \brief Converts a weight stored in a synapse row to an input
+////! \param[in] weight: the weight to convert in synapse-row form
+////! \param[in] left_shift: the shift to use when decoding
+////! \return the actual input weight for the model
+//static inline input_t synapse_row_convert_weight_to_input(
```

Comment:
Are there future plans to use this new and commented out code

Reply:
I think this is *old* commented out code rather than new since it has a left-shift in it.  I'm not sure how it stayed in but perhaps there was some confusion during a merge and it was kept somehow.

Reply:
Thinking about it further I think this "conversion" was moved from here during the split neuron-synapse modelling, so it was probably when that was merged into this branch that the confusion happened.